src/js/grapl-cdk/lib/alarms.ts

```diff
@@ -54,7 +54,7 @@ class RiskNodeAlarm extends cdk.Construct {
             metricName: 'risk_node',
             dimensions: {},
         });
```

Comment:
This change is not needed - it was an attempt to include the Alarm in the CDK stack, but it doesn't...